src/rules/lambda/LambdaLatestVersion.ts

```diff
@@ -0,0 +1,74 @@
+/*
+Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+SPDX-License-Identifier: Apache-2.0
+*/
+import { parse } from 'path';
+import { CfnFunction, Runtime } from '@aws-cdk/aws-lambda';
+import { CfnResource } from '@aws-cdk/core';
+import { NagRuleCompliance, NagRules } from '../../nag-rules';
+
+/**
+ * Lambda functions are configured to use the latest runtime version
+ * @param node the CfnResource to check
+ */
+export default Object.defineProperty(
+  (node: CfnResource): NagRuleCompliance => {
+    if (node instanceof CfnFunction) {
+      const runtime = NagRules.resolveIfPrimitive(node, node.runtime);
+      if (!runtime) {
+        return NagRuleCompliance.NON_COMPLIANT;
```

Comment:
This would mean that any [container image](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-resource-lambda-function.html#cfn-lambda-function-packagetype) lambda function would be marked as non-compliant, which I don't think is the intention of this rule.

I'm that case we might want to update the description of the rule to say that it only applies to non-container lambda functions